nextflow_schema.json

```diff
@@ -229,7 +229,7 @@
                 "prokka_batchsize": {
                     "type": "string",
                     "description": "Size of individual files annotated by Prokka in one batch.",
-                    "default": "10.MB",
+                    "default": "10 MB",
```

Comment:
Is this correct? On the command line you can't say `10 MB` without using quotes.

Reply:
I think it's a change made by prettier or lint, I didn't change it. Anyway I didn't get any warning about it.